ivy/functional/ivy/general.py

```diff
@@ -2088,13 +2088,25 @@ def get_tmp_dir():
 
 
 @handle_exceptions
-def set_tmp_dir(tmp_dr):
+def set_tmp_dir(tmp_dr: str) -> None:
     """Set the directory for saving temporary files.
 
     Parameters
     ----------
     tmp_dr
+        The new directory for saving temporary files
+    Examples
+    --------
+    >>> x = ivy.get_tmp_dir()
+    >>> print(x)
+    /tmp
 
+    To set the temp base to 1e-04
+
+    >>> ivy.set_tmp_dir('/my_tmp')
```

Comment:
```suggestion
    >>> ivy.set_tmp_dir("/my_tmp")
```
Just putting strings in double quotes  as normal python string representation